src/components/tag/tag.js

```diff
@@ -283,10 +284,6 @@ const Tag = props => (
               `,
               props.isDisabled &&
                 css`
-                  &:hover {
```

Comment:
This isn't needed because when a button is disabled in the HTML, the `hover` and `focus` states no longer apply.